pkg/helm3/schema/schema.json

```diff
@@ -62,6 +62,10 @@
                 "type":"string"
               }
             },
+            "atomic": {
```

Comment:
Can you add a description property as well? When it's set, users get a nice inline doc popup when editing a porter.yaml file in VS Code so it's helpful to include.